### PR TITLE
Add/app marionette js

### PR DESCRIPTION
**Pull request by avasuro**
Issue: #7980

## What I did
Added support of Marionette.js framework

## How to test

- Is this testable with Jest or Chromatic screenshots? - don't know, done similar as html app
- Does this need a new example in the kitchen sink apps? - don't know
- Does this need an update to the documentation? - done (I think)

### CODE REVIEW

**Comment by shilman** [comment]
Great work @avasuro -- very exciting! We're just getting 5.2 out the door, so we'll have to wait until that's released to merge this.

**Comment by avasuro** [comment]
@shilman, thank you! no problem.

File: README.md [hunk 1]
```diff
@@ -115,6 +115,7 @@ For additional help, join us [in our Discord](https://discord.gg/sMFvFsG) or [Sl
 | [Vue](app/vue)                   | [v5.1.0](https://storybooks-vue.netlify.com/)     | [![Vue](https://img.shields.io/npm/dm/@storybook/vue.svg)](app/vue)                            |
 | [Angular](app/angular)           | [v5.1.0](https://storybooks-angular.netlify.com/) | [![Angular](https://img.shields.io/npm/dm/@storybook/angular.svg)](app/angular)                |
 | [Polymer](app/polymer)           | [v5.1.0](https://storybooks-polymer.netlify.com/) | [![Polymer](https://img.shields.io/npm/dm/@storybook/polymer.svg)](app/polymer)                |
+| [Marionette.js](app/marionette)  | -                                                 | [![Mithril](https://img.shields.io/npm/dm/@storybook/marionette.svg)](app/marionette)                |
```
Comment (ndelangen) [inline README.md]: Says "Mithril" as alt-text

**Review by ndelangen — APPROVED** [review]
Do we need to add a example for this?

**Comment by avasuro** [comment]
I think yes, for consistency (all other frameworks except react native has examples). But I didn't figured out how to do that.

**Comment by ndelangen** [comment]
@avasuro shall we schedule a pair programming session to finish this?

contact me on discord:
https://discord.gg/sMFvFsG

We need to add an example of a marionette app that has storybook.

**Comment by ndelangen** [comment]
@avasuro do you think you'd be available to work together on this to get this merged?

**Comment by avasuro** [comment]
@ndelangen 

Hi, yes. I will write you in discord.

**Review by ndelangen — APPROVED** [review]

**Pull request merged**